samples/nrf5340/netboot/README.rst

```diff
@@ -7,82 +7,99 @@ nRF5340: Network core bootloader
-     Once the copy is done, compare the SHA of the data in the application partition against the SHA specified in the shared SRAM.
-     Communicate the result of the comparison to the application core through the shared SRAM.
+#. Performing any pending network core firmware update.
+     The sample calls the :ref:`subsys_pcd` library to inspect the SRAM region shared with the application core:
 
-#. Lock the flash of the application.
-     Lock the flash that contains the application.
+     a. If the application core has written an update instruction, the bootloader copies the specified data range to the application partition on the network core.
+     #. Once the copy is done, the sample compares the SHA of the data in the application partition against the SHA specified in the shared SRAM.
+     #. It then communicates the result of the comparison to the application core through the shared SRAM.
+
+#. Locking the flash memory areas containing the application.
+     The sample locks the flash memory areas that contain the application.
      Locking is done using the ACL peripheral.
      For details on locking, see the :ref:`fprotect_readme` driver.
 
-#. Boot the application on the network core.
-     After possibly performing a firmware update, and enabling flash protection, the network core bootloader uninitializes all peripherals that it used and boots the application.
+#. Booting the application on the network core.
+     After possibly performing a firmware update, and enabling flash memory protection, the network core bootloader uninitializes all peripherals that it used and boots the application.
+
+The most common use case for the network core bootloader sample is to be included as a child image in a multi-image build, rather than being built stand-alone.
 
 Requirements
 ************
 
-  * |nRF5340DK|
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340dk_nrf5340_cpunet
 
 .. _net_bootloader_build_and_run:
 
 Building and running
 ********************
 
-The source code of the sample can be found under :file:`samples/nrf5340/netboot/` in the |NCS| folder structure.
+The recommended way of building the sample is to use the :ref:`multi-image feature <ug_multi_image>` of the build system.
 
-The most common use case for the network core bootloader sample is to be included as a child image in a multi-image build, rather than being built stand-alone.
-This sample is included automatically if the application in the nRF5340 network core has the :option:`CONFIG_SECURE_BOOT` option set.
-Note that the build system will automatically enable the :option:`CONFIG_SECURE_BOOT` option for the network core application if the application on the application core has enabled :option:`CONFIG_BOOTLOADER_MCUBOOT` and :option:`CONFIG_BT_RPMSG_NRF53`.
+You can include the sample in your multi-image build in two different ways:
```

Comment:
How do you choose which option to use?
It seems to depend on if you have an application on the application core, or if you want to include the bootloader in the application from the network core or the application core ...?

Once clarified, it might be clearer to use a definition list than a bullet list, so something like:

```
To include the bootloader from the network core application
   Enable ...

To include the bootloader from the application core application
   Enable ...
```

Reply:
@ru-fu I rephrased the section, please check if you still think that a definition list might be better than a bullet list.

Reply:
outdated